test/integration/fixtures/pending/skip-async-before-nested.fixture.js

```diff
@@ -0,0 +1,35 @@
+describe('skip in before with nested describes', function () {
+  before(function (done) {
+    var self = this;
+    setTimeout(function () {
```

Comment:
what's the necessity of doing this asynchronously?

Reply:
@boneskull I'm not sure what the "real" use case would look like, but this test validates that an asynchronous skip works with a synchronous nested `describe`. It seemed like something worth checking, even as an edge case. 🤷‍♀️

Reply:
Many of these `setTimeout()`-based tests are simply race conditions that happen to have worked in @bannmoore's favor. Shouldn't we just disallow use of async `this.skip()`? Otherwise, the results are not deterministic...